flask_monitoringdashboard/core/config/__init__.py

```diff
@@ -41,6 +41,17 @@ def __init__(self):
         self.guest_password = ['guest_password']
         self.security_token = 'cc83733cb0af8b884ff6577086b87909'
 
+        try:
+            self.host_name = os.environ['CONTAINER_NAME']
```

Comment:
use `os.environ.get(name, default)` or `if name in os.environ` instead of try-except everything